### ISSUE TITLE
[BUG] Insufficient Color Contrast in Azure Boards config dialog

### ISSUE THREAD

**Issue by DaveTryon** [microsoft/accessibility-insights-windows#601]
Please check whether the bug has [already been filed](https://github.com/Microsoft/accessibility-insights-windows/issues).

**Describe the bug**
The placeholder text for the Azure Boards link uses a fixed color value of SlateGray, which is #708090. When rendered on a white background (the default), the color contrast is only 4.054:1, which is insufficient for 12px text. It also fails to respond to High Contrast modes because it's a hardcoded color.

**To Reproduce**
1. Go to Azure Boards configuration page
2. Look at the placeholder text in the link control in various HC modes (Off, White, Black)
3. Text is exceptionally faint (even for placeholder text)

**Expected behavior**
Text should satisfy color contrast requirements normally, and become sharper in HC modes

**Actual behavior**
Contrasts are as follows:
HC Off: 708090 on FFFFFF: Ratio is 4.054:1
HC White: 708090 on FFFFFF: Ratio of 4.054:1
HC 1, 2, or Black: 708090 o FFFFFF: Ratio of 5.179:1

**Screenshots or .GIF**
Control in various modes:
![image](https://user-images.githubusercontent.com/45672944/66789339-fe968980-ee9f-11e9-9565-d0c992a3e874.png)

**Desktop (please complete the following information):**
 - OS: Windows 10 1909
 - Accessibility Insights for Windows Version: 1.1.1013.2
 - Target Application: n/a
 - Target Application Version: n/a

**Additional context**
Priority requested -
